cpp/ql/test/experimental/query-tests/Security/CWE/CWE-295/CurlSSL.qlref

```diff
@@ -0,0 +1 @@
+experimental/Security/CWE/CWE-295/CurlSSL.ql
```

Comment:
There should also be a `.expected` file for the test, showing the results we get from running the query.  Let us know if you need more help with this.